cassiopeia/solver/dissimilarity_functions.py

```diff
@@ -137,39 +137,11 @@ def hamming_distance(s1: np.array(int), s2: np.array(int)) -> int:
     return dist
```

Comment:
All the other functions here return a score normalized by the number of non-missing characters. Do we want that behavior here? 

Or we can add an argument requesting the function normalize by the number of non-missing entries, or not?

Reply:
Given that this is only used in the calculation of LCA distances in the potential graph for ILP, maybe we shouldn't normalize it?

Reply:
I agree, let's not normalize it here. It's much easier to normalize after the fact than to un-normalize something.